### ISSUE TITLE
What is the JHIPSTER_REGISTRY_PASSWORD used for? 

### ISSUE THREAD

**Issue by nmarkon** [jhipster/jhipster-registry#577]
##### **Overview of the issue**

I'm tasked with upgrading the version of the jhipster registry we're using, from 6.8 to 7.4. As part of this process I'm trying to document how we have it configured. 

We currently have both the setup:

```
- SPRING_SECURITY_USER_PASSWORD=xxx
- JHIPSTER_REGISTRY_PASSWORD=same as JHIPSTER_REGISTRY_PASSWORD
- JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET=encodedstring
```

I see the SPRING_SECURITY_USER_PASSWORD, and JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET explained in the documentation, but I can't find what JHIPSTER_REGISTRY_PASSWORD is for. Maybe I'm missing it..  Any help? I think it's the password the apps used to authenticate with registry, but I'd like to confirm that.

**Comment by jdubois** [comment]
Yes, the JHipster Regitry is secured by default, so the apps need this password to connect to it. Otherwise you’ll see authentication issues in the client apps, so you’ll see the error quickly.
